public/app/features/dashboard-scene/scene/PanelMenuBehavior.tsx

```diff
@@ -57,12 +56,8 @@ export function panelMenuBehavior(menu: VizPanelMenu) {
           text: t('panel.header-menu.edit', `Edit`),
           iconClassName: 'eye',
           shortcut: 'e',
-          onClick: () => () => DashboardInteractions.panelMenuItemClicked('edit'),
-          href: getDashboardUrl({
-            uid: dashboard.state.uid,
-            subPath: `/panel-edit/${panelId}`,
-            currentQueryParams: location.search,
-          }),
+          onClick: () => DashboardInteractions.panelMenuItemClicked('edit'),
```

Comment:
can we still have this be a URL? it's so nice to support CTRL+click

Reply:
@torkelo  it's still an URL, see line below :)

Reply:
Ah, nisses it